ons_alpha/jinja2/templates/components/streamfield/headline_figures_block.html

```diff
@@ -12,6 +12,12 @@ <h3 class="headline-figures__title">
                         </a>
                     {% endif %}
                 </h3>
+                {% if figure.image %}
+                    <figure class="headline-figures__image-container">
+                        {{ srcset_image(figure.image, "format-png|width-{278,556}", class="headline-figures__image") }}
```

Comment:
I chose `format-png` as the example images show diagrams with text, and a lossless format would be more appropriate here to preserve the content.